force_wfmanager/left_side_pane/mco_parameter_model_view.py

```diff
@@ -0,0 +1,45 @@
+from traits.api import Instance, Str, on_trait_change
+
+from traitsui.api import View, Item, ModelView
+
+from force_bdss.api import BaseMCOParameter
+
+from .view_utils import get_factory_name
+
+
+class MCOParameterModelView(ModelView):
+    #: MCO parameter model
+    model = Instance(BaseMCOParameter, allow_none=False)
```

Comment:
when you define a modelview, there's an entry called "object" that refers to your model automatically. You need to check the modelview api.